.github/workflows/integration-testing.yaml

```diff
@@ -188,7 +189,33 @@ jobs:
       - name: Wait for env
         run: while [[ $( curl -s -o /dev/null -w "%{http_code}" "nginx-integration-${{ needs.variables.outputs.commit }}.${{secrets.DEV_SANDBOX_NAMESPACE}}/texts") != "200" ]]; do sleep 10; done
         timeout-minutes: 5
-      - name: Run tests
-        run: if [[ $( curl -s -o /dev/null -w "%{http_code}" "nginx-integration-${{ needs.variables.outputs.commit }}.${{secrets.DEV_SANDBOX_NAMESPACE}}/texts") != "200" ]]; then exit 1; fi
+
+      - name: Set SANDBOX_URL
+        run: echo "SANDBOX_URL=http://nginx-integration-${{ needs.variables.outputs.commit }}.${{ secrets.DEV_SANDBOX_NAMESPACE }}/texts" >> $GITHUB_ENV
+      
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

Comment:
This is a little bit old - github is in the process of upgrading all runners to node 20 by default at the moment, actions that expect node to be present will start to expect that soon.  It might also be worth baking node into the runner image, save some time during testing

Reply:
makes sense - what do we need to do in order to get it into the image? that would save us a lot of time